_inc/client/components/jetpack-termination-dialog/features.jsx

```diff
@@ -134,6 +147,17 @@ class JetpackTerminationDialogFeatures extends Component {
 						</ul>
 					</div>
 				) }
+				{ connectedPlugins && (
+					<div className="jetpack-termination-dialog__generic-info">
+						<h2>{ __( 'Jetpack Connection is also used by these plugins' ) }</h2>
+						<p>
+							{ __(
+								'To fully disconnect form Jetpack you will need to disable following plugins:'
```

Comment:
I'd recommend something like this:

```suggestion
								'To fully disconnect your site from WordPress.com, you will need to deactivate the following plugins:'
```

Reply:
Nicely said, done :)